src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -2181,6 +2190,28 @@ int sd_dhcp_client_stop(sd_dhcp_client *client) {
         return 0;
 }
 
+int sd_dhcp_client_initialize_timeout_ipv6_only_preferred(sd_dhcp_client *client, uint32_t delay) {
```

Comment:
Please always suffix `_sec` for timespan (or timestamp) whose unit is second.
Also, rename the function to `_ipv6_only_mode()`.